frame/support/src/storage/bounded_vec.rs

```diff
@@ -391,7 +285,7 @@ where
 	BoundedVec<T, S>: Encode,
 {
 	fn max_encoded_len() -> usize {
-		// BoundedVec<T, S> encodes like Vec<T> which encodes like [T], which is a compact u32
+		// WeakBoundedVec<T, S> encodes like Vec<T> which encodes like [T], which is a compact u32
```

Comment:
You've changed the comment, but this is still the implementation for `BoundedVec`.